fe/fe-core/src/main/java/com/starrocks/sql/optimizer/cost/CostModel.java

```diff
@@ -126,6 +129,25 @@ public CostEstimate visitOperator(Operator node, ExpressionContext context) {
             return CostEstimate.zero();
         }
 
+        private CostEstimate adjustCostForMV(ExpressionContext context) {
```

Comment:
if the base table is olap table, is adjusting cost for MV reasonable?

Reply:
Yes, if base table does not have stats, but mv has, it will choose mv operator